index.html

```diff
@@ -36,3 +34,155 @@ <h3 class="intro">I'm Bunbee<br>Glad to see<br> you!</h3>
         <li class="fa fa-facebook fa-2x"></li>
        </ul>
     </section>
+
+    <section class="card-section">
```

Comment:
- [ ] Great job adding the display grid to the card-section, however, as mentioned by the previous reviewer the `display: grid;` should be only available on the main container of the projects (container of all the projects). for now, all the projects have the `display: grid;`, we should have one main container to wrap all the projects and that is the container that should display as a grid, but individual project just only have a `<div>` as a container and should display flex instead of grid. this means we should only have the `display: grid;` once on the projects container 👍